README.md

```diff
@@ -9,16 +9,40 @@ maintained by Interchain GmbH,
 and one in [Rust](https://github.com/informalsystems/tendermint-rs),
 maintained by Informal Systems.
 
-There have been inadvertent divergences in the specs followed
-by the Go implementation and the Rust implementation respectively.
-However, we are working toward reconverging these specs into a single unified spec.
-Consequently, this repository is in a bit of a state of flux.
+### Data Structures
 
-At the moment, the spec followed by the Go implementation
-(tendermint/tendermint) is in the [spec](spec) directory,
-while the spec followed by the Rust implementation
-(informalsystems/tendermint-rs) is in the [rust-spec](rust-spec)
-directory. TLA+ specifications are also in the rust-spec directory.
+- [Encoding and Digests](./spec/core/encoding.md)
+- [Blockchain](./spec/core/data_structures.md)
+- [State](./spec/core/state.md)
```

Comment:
nit: I notice some subsections have a short description following the link and others don't. It would be nice although not necessary if this was consistent.